Dockerfile

```diff
@@ -1,13 +1,12 @@
 # Keep hub.Dockerfile aligned to this file as far as possible
 ARG JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom"
-ARG APP_INSIGHTS_AGENT_VERSION=2.3.1-SNAPSHOT
+ARG APP_INSIGHTS_AGENT_VERSION=2.5.1-BETA
 
-FROM hmctspublic.azurecr.io/base/java:openjdk-8-distroless-1.0
+FROM hmctspublic.azurecr.io/base/java:openjdk-8-distroless-1.2
 LABEL maintainer="https://github.com/hmcts/ccd-user-profile-api"
 
 COPY build/libs/user-profile.jar  /opt/app/
 COPY lib/AI-Agent.xml /opt/app
-COPY lib/applicationinsights-agent-2.3.1-SNAPSHOT.jar lib/AI-Agent.xml /opt/app/
```

Comment:
you added applicationinsights-agent-2.5.1-BETA.jar. Is this required? Isn't this automatically downloaded by the base image?

Reply:
confirmed with Tim. That is correct, but we still need it there because it's needed in the ASE envs.

Reply:
Still need this for  ASE, confirmed from Praveen